backend/tournesol/views/comparison.py

```diff
@@ -117,8 +117,10 @@ def perform_create(self, serializer):
         comparison: Comparison = serializer.save()
         comparison.entity_1.update_n_ratings()
         comparison.entity_1.inner.refresh_metadata()
+        comparison.entity_1.auto_remove_from_rate_later(self.request.user)
         comparison.entity_2.update_n_ratings()
         comparison.entity_2.inner.refresh_metadata()
+        comparison.entity_2.auto_remove_from_rate_later(self.request.user)
```

Comment:
`auto_remove_from_rate_later()` is currently specific to the poll "videos" and relies on the model `VideoRateLater`. Should we add a condition on the poll name here (or inside the method) to avoid unnecessary checks for other polls?

Reply:
It currently works because `VideoRateLater.video` is a foreign key to `Entity` so any entity could be removed from it.

VideoRateLater seems very easy to convert into EntityRateLater and might be a useful feature for other polls? What do you think of instead of adding the condition here, we'd fully migrate VideoRateLater to Entities? Needs not much more than to add a new field `poll`.

Reply:
Or, are you suggesting it is not desirable because it hinders performance unnecessarily?

Reply:
> VideoRateLater seems very easy to convert into EntityRateLater and might be a useful feature for other polls?

Yes I think the rate-later list could be relevant for any poll.

> What do you think of instead of adding the condition here, we'd fully migrate VideoRateLater to Entities?

I like this idea, it will make the code base even more clean by removing specific code that is not that specific. It will also implies to move the endpoint rate later to a poll-scoped endpoint. (can be done in several PR.)